smt/surrogate_models/krg_based.py

```diff
@@ -833,18 +883,28 @@ def grad_minus_reduced_likelihood_function(log10t):
             k, incr, stop, best_optimal_rlf_value, max_retry = 0, 0, 1, -1e20, 10
             while k < stop:
                 # Use specified starting point as first guess
-                if self.options["eval_noise"]:
+                if self.options["eval_noise"] and not self.options["is_noise_het"]:
```

Comment:
Same remark

Reply:
In this initial implementation, the noise variances are pointwise sensible estimates via (Ankenman et al., 2010), and they are not estimated via ML, i.e. they are fixed for the MLE. In future developments, they will be integrated in the MLE.